README.md

```diff
@@ -26,8 +26,12 @@ var params = {
   keypairId: process.env.PUBLIC_KEY,
   privateKeyString: process.env.PRIVATE_KEY,
   privateKeyPath: '/path/to/private/key',      // Optional. Use as an alternative to privateKeyString.
-  expireTime: '<epoch time when you wish the link to expire>'
+  expireTime: new Date(new Date().getTime() + (1000 * 100)) // Now + 100 seconds
```

Comment:
I still don't really get the value add of the js Date object. You're instantiating a new Date object to get the time in ms, then using the result to instantiate another Date object, only to convert back to ms again later.

How is 
```js
{expireTime: new Date(new Date().getTime() + (1000 * 100))}
```
more straightforward than
```js
{expireTime: new Date().getTime() + (1000 * 100)}
```
Also, passing the time as ms in the first place eliminates a lot of unnecessary calls to `.getTime` throughout the rest of the code.